### DISCUSSION TITLE
menu with joystick/gamepad

### DISCUSSION THREAD

**Discussion by JonecoBoy** [mikke89/RmlUi#513] [Q&A]
hi am going to choose a UI lib for my emulators frontend project and i am wondering if RmlUI can work with joypads/ gamepads. i need any other lib? any sugestion? thanks

**Comment by brunoalef7** [comment]
You need to write your own code to interact gamepads with RmlUI, i recommend you to use SDL to handle the inputs. 
I'm currently working on a game and I needed to create a separate lib to use gamepad in the game interface (RmlUi)
